docs/antora/modules/ROOT/pages/Getting_Started.adoc

```diff
@@ -39,78 +41,66 @@ out/
         ...
 ----
 
-You can download an example project with this layout here:
-
-* {mill-github-url}/releases/download/{mill-last-tag}/{mill-last-tag}-example-1.zip[Example 1]
```

Comment:
@lefou what is this syntax? I tried to paste it into https://asciidoclive.com/ and it does not render as a link. Is this from some kind of asciidoc syntax extension or something?

Reply:
The inside of the curly braches are variables.

```
$ rg mill-github-url
build.sc
1641:         |    mill-github-url: ${Settings.projectUrl}
```

Reply:
For testing in the live url, you can define a section followed by variable definitions, e.g.

```adoc
== test section
:mill-github-url: TEST-URL
:mill-last-tag: 999

* {mill-github-url}/releases/download/{mill-last-tag}/{mill-last-tag}-example-1.zip[Example 1]
```